packages/store-ui/src/atoms/Overlay/stories/Overlay.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import type { Story, Meta } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { Props as OverlayProps } from '../Overlay'
+import Component from '../Overlay'
+import mdx from './Overlay.mdx'
+
+type ColorType = 'black' | 'green'
+type OverlayStoryProps = OverlayProps & { color: ColorType }
+
+const OverlayTemplate: Story<OverlayStoryProps> = ({
+  color = 'black',
+  ...props
+}) => {
+  const styleMap = {
+    green: {
+      backgroundColor: 'rgba(0, 255, 0, 0.5)',
+    },
+    black: {
+      backgroundColor: 'rgba(0, 0, 0, 0.5)',
+    },
+  }
+
+  return (
+    <div style={{ height: 'calc(100vh + 150px)' }}>
+      A content outside me
+      <Component
+        {...props}
+        style={{
```

Comment:
Nice! I think we can wait for #828 so we can move these styles there, what do you think?